nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -1025,11 +1060,20 @@ private Request configureRequest(final ProcessContext context, final ProcessSess
 
     private RequestBody getRequestBodyToSend(final ProcessSession session, final ProcessContext context, final FlowFile requestFlowFile) {
         if(context.getProperty(PROP_SEND_BODY).asBoolean()) {
-            return new RequestBody() {
+            String evalContentType = context.getProperty(PROP_CONTENT_TYPE)
+                .evaluateAttributeExpressions(requestFlowFile).getValue();
+            final String contentType = StringUtils.isBlank(evalContentType) ? DEFAULT_CONTENT_TYPE : evalContentType;
+            HashMap<String,PropertyDescriptor> propertyDescriptors = new HashMap<>();
```

Comment:
Best to declare as a `Map<String, PropertyDescriptor>` rather than `HashMap`

Reply:
yikes, good point. done